src/ElmHtml/InternalTypes.elm

```diff
@@ -6,6 +6,7 @@ module ElmHtml.InternalTypes
         , CustomNodeRecord
         , MarkdownNodeRecord
         , Facts
+        , Tagger
```

Comment:
Right, because it's just an alias, but it will bump the version anyways because Facts is exposed and was changed to include the taggers

But I can remove this alias if you want